### PR TITLE
Fix/tablet design

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Solved design issues. 
2) Added spacing related to large screens.
3) Solved UI crash bug on release build .
4) Image not pick widget is converted into dialog for tablets and mobile.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -12,11 +12,16 @@ import 'package:get_it/get_it.dart';
...
 
+
+bool isTablet = false;
 void main() {
   WidgetsFlutterBinding.ensureInitialized();
   PylonsWallet.setup(mode: PylonsMode.prod, host: 'easel');
   di.init();
 
+
+  isTablet = MediaQueryData.fromWindow(WidgetsBinding.instance!.window).size.shortestSide >= 600;
```
Comment (mustafapylons) [inline lib/main.dart]: "600" should be in a constant.

**Review by mustafapylons — COMMENTED** [review]
One small comment, otherwise code is good.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Review by kiroj305 — APPROVED** [review]
Added one more design fix in `tutorial 3` screen.

**Pull request merged**